README.md

```diff
@@ -64,6 +64,8 @@ Note that the external-provisioner does not scale with more replicas. Only one e
 #### Other recognized arguments
 * `--feature-gates <gates>`: A set of comma separated `<feature-name>=<true|false>` pairs that describe feature gates for alpha/experimental features. See [list of features](#feature-status) or `--help` output for list of recognized features. Example: `--feature-gates Topology=true` to enable Topology feature that's disabled by default.
 
+* `--strict-topology`: This controls what topology information is passed to `CreateVolumeRequest.AccessibilityRequirements` in case of delayed binding. Use this option to pass only selected node topology to `AccessibilityRequirements` instead of aggregated cluster topology. This option has no effect either `Topology` feature is disabled or `Immediate` volume binding mode is used.
```

Comment:
Instead of the "Use this option..." sentence I would just say:
```
See [the table below](#topology-support) for an explanation how this option changes the result.
```

That's because this shorthand description doesn't fully explain the effect and trying to make it complete could become lengthy. The table is much nicer.

Missing "if" in "no effect *if* either".

Reply:
Made suggested changes.